src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,36 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_SIZE = 10;
+    private Object[] objectsK = new Object[MAX_SIZE];
+    private Object[] objectsV = new Object[MAX_SIZE];
```

Comment:
```suggestion
    private Object[] keys = new Object[MAX_SIZE];
    private Object[] values = new Object[MAX_SIZE];
```
https://mate-academy.github.io/jv-program-common-mistakes/java-core/generics/storage.html#all-non-constant-fields-should-be-initialized-in-the-constructor
> All non-constant fields should be initialized in the constructor